lib/Conversion/TritonGPUToLLVM/DotOpToLLVM/FMADotUtility.cpp

```diff
@@ -103,11 +103,16 @@ LogicalResult parametricConvertFMADot(DotOp op, DotOp::Adaptor adaptor,
   Value llA = adaptor.getA();
   Value llB = adaptor.getB();
 
-  auto sizePerThread =
-      expandMatrixShapeWithBatch(ArrayRef(getSizePerThread(dLayout)));
+  auto sizePerThread = getContigPerThread(dTensorTy);
```

Comment:
`getContigPerThread` and `getSizePerThread` are implemented differently in linear layout

Reply:
they are indeed. `getSizePerThread` does not make much sense tho, so I'm trying to use it as little as possible